docs/application/dotnet/guides/location-sensors/device-sensors.md

```diff
@@ -291,9 +297,62 @@ The following table lists the measurement data that the HRM sensor provides.
 
 | Measurement | Type                 | Range                          | Unit         |
 |-----------|--------------------|------------------------------|------------|
+| Timestamp  | `ulong`    | -                              | Microseconds |
 | TimeSpan   | `TimeSpan` | -                              | Microseconds |
 | HeartRate   | `int`                | Min. value = 0<br>Max. value = 240 | -            |
 
+<a name="hrm_batch"></a>
+## Heart Rate Monitor Batch Sensor
+
+The Heart Rate Monitor (HRM) batch sensor measures a person's heart rate. The sensor must be power-efficient and support batch processing.
+
+The following table lists the measurement data that the HRM batch sensor provides:
```

Comment:
It is recommended to use a colon at the end of the sentence that uses "following", so this sentence is fine. However, throughout the file, I see that in many places, a period is used instead of a colon for the same scenario. So there can be inconsistency issues on this page. 

@mijin85cho Kindly suggest how to proceed in this situation.

@bwikbs Please do not change anything, we will wait for @mijin85cho's suggestion.

Same comment at Line no. 334